tests/test.rs

```diff
@@ -5,6 +5,7 @@ fn no_items() {
     let grid = Grid::new(GridOptions {
         direction: Direction::TopToBottom,
         filling: Filling::Spaces(2),
+        ..Default::default()
```

Comment:
seems that you are changing the usage of the API by making default mandatory.
I would prefer we don't have to do that.